domains/auth/Auth.API/API/Controllers/InvalidateController.cs

```diff
@@ -14,17 +12,17 @@ public class InvalidateController : ControllerBase
 {
     private readonly IOidcService oidcService;
     private readonly IValidator<AuthState?> validator;
-    private readonly ICryptography cryptography;
+    private readonly ICryptography stateCryptography;
 
     public InvalidateController(
         IOidcService oidcService,
-        ICryptography cryptography,
-        IValidator<AuthState> validator
+        ICryptography stateCryptography,
```

Comment:
Is this working? i would believe it should be the factory you inject here so on line `25` it would look something like this:
`this.stateCryptography = cryptro.StateCryptography();`